tests/apollo/util/skvbc.py

```diff
@@ -221,6 +221,20 @@ async def assert_kv_write_executed(self, key, val):
             assert {key: val} == kv_reply, \
                 f'Could not read original key-value in the case of n={config.n}, f={config.f}, c={config.c}.'
 
+    async def wait_for_liveness(self):
+        with trio.fail_after(seconds=30):
+            while True:
+                with trio.move_on_after(seconds=.5):
```

Comment:
As suggested by @KTeofanidis , it should be `seconds=5` here (otherwise it is very likely to interrupt the await for the client write before it has a chance to complete).